src/data/reference/hi.json

```diff
@@ -606,53 +627,87 @@
+                "<a href=\"#/p5/push\">push()</a> निम्नलिखित कार्यों द्वारा नियंत्रित वर्तमान परिवर्तन स्थिति और शैली सेटिंग्स से संबंधित जानकारी संग्रहीत करता है: <a href=\"#/p5/fill \">भरें()</a>, <a href=\"#/p5/noFill\">noFill()</a>, <a href=\"#/p5/noStroke\">noStroke() </a>, <a href=\"#/p5/stroke\">stroke()</a>, <a href=\"#/p5/tint\">tint()</a>, <a href=\"#/p5/noTint\">noTint()</a>, <a href=\"#/p5/strokeWeight\">strokeWeight()</a>, <a href=\"# /p5/strokeCap\">strokeCap()</a>, <a href=\"#/p5/strokeJoin\">strokeJoin()</a>, <a href=\"#/p5/imageMode\" >imageMode()</a>, <a href=\"#/p5/rectMode\">rectMode()</a>, <a href=\"#/p5/ellipseMode\">ellipseMode()</a>, <a href=\"#/p5/colorMode\">colorMode()</a>, <a href=\"#/p5/textAlign\">textAlign()</a>, <a href =\"#/p5/textFont\">textFont()</a>, <a href=\"#/p5/textSize\">textSize()</a>, <a href=\"#/p5 /textLeading\">textLeading()</a>, <a href=\"#/p5/applyMatrix\">applyMatrix()</a>, <a href=\"#/p5/resetMatrix\">resetMatrix ()</a>, <a href=\"#/p5/rotate\">rotate()</a>, <a href=\"#/p5/scale\">scale()</a>, <a href=\"#/p5/shearX\">shearX()</a>, <a href=\"#/p5/shearY\">shearY()</a>, <a href=\"#/p5/translate\">translate()</a>, <a href=\" #/p5/noiseSeed\">noiseSeed()</a>. ",
+                "WEBGL मोड में अतिरिक्त स्टाइल सेटिंग्स संग्रहीत की जाती हैं। इन्हें निम्नलिखित कार्यों द्वारा नियंत्रित किया जाता है: <a href=\"#/p5/setCamera\">setCamera()</a>, <a href=\"#/p5/ ambientLight\">ambientLight ()</a>, <a href=\"#/p5/directionLight\">directionLight()</a>, <a href=\"#/p5/pointLight\">pointLight()</a>, <a href=\"#/p5/texture\">texture()</a>, <a href=\"#/p5/specularMaterial\">specularMaterial()</a>, <a href=\"#/p5/shininess\">shininess()</a>, <a href=\"#/p5/normalMaterial\">normalMaterial()</a> और <a href=\" #/p5/shader\">shader()</a>।"
+              ],
             "returns": "p5 ऑब्जेक्ट"
         },
         "pop": {
-            "description": "पुश () फ़ंक्शन वर्तमान ड्राइंग शैली सेटिंग को रिकॉर्ड करता है, और पॉप () इस सेटिंग को पुनर्स्थापित करता है। ध्यान दें कि ये फ़ंक्शन हमेशा एक साथ उपयोग किए जाते हैं। वे आपको शैली सेटिंग्स और परिवर्तनों को बदलने की अनुमति देते हैं और फिर जो आपके पास था, उसे वापस करते हैं। जब एक नया राज्य पुश () के साथ शुरू किया जाता है, तो यह वर्तमान शैली और परिवर्तन जानकारी पर बनाता है। अधिक नियंत्रण प्रदान करने के लिए पुश () और पॉप () फ़ंक्शन को एम्बेड किया जा सकता है (प्रदर्शन के लिए दूसरा उदाहरण देखें)। पुश () वर्तमान स्थिति और परिवर्तन राज्य सेटिंग्स से संबंधित जानकारी संग्रहीत करता है, जो निम्न कार्यों द्वारा नियंत्रित होता है: भरण (), स्ट्रोक (), टिंट (), स्ट्रोक वाइट (), स्ट्रोककैप (), स्ट्रोकजोन (), इमेजमोड () ), rectMode (), ellipseMode (), colorMode (), textAlign (), textFont (), textMode (), textSize (), textLeading ()।"
```

Comment:
**Please Don't translate name of functions, variables, code snippet, etc.**

Reply:
Yes, I am also trying to do the same.